apps/chainlit/app.py

```
@@ -1872,38 +2083,125 @@ class RegisterRequest(BaseModel):
-        return {"message": "Registrierung erfolgreich", "username": user["identifier"]}
+        if EMAIL_VERIFICATION_ENABLED and SMTP_HOST:
+            token = secrets.token_urlsafe(32)
+            user = await create_user(
+                DATABASE_URL, username, email, password_hash,
+                email_verified=False, verification_token=token,
+            )
+            if user is None:
+                raise HTTPException(status_code=500, detail="Registrierung fehlgeschlagen")
+            try:
+                await _send_verification_email(email, username, token)
+            except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)